lab/skel_graph/traversal/problems/alee/alee.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020 Nicolae Mara-Ioana <maraioana9967@gmail.com>
+
+// Alee - Drumuri minime in graf
+// Complexitate: O(n * m)
+
+// https://www.infoarena.ro/problema/alee
+
+#include <bits/stdc++.h>
+
+class Task {
+	
+ public:
+	
+	void solve() {
+		read_input();
+		print_output(get_result());
+	
+	}
+
+ private:
+	int n;
+	int m;
+	int source;
+	std::vector<std::vector<char>> graph;
+    std::pair<int, int> startGate;
+    std::pair<int, int> endGate;
+	
+    // s-a ales urmatoarea codificare:
+    // graph[i][j] = 'T', atunci exista un copcac la coordonatele i, j
+    // graph[i][j] = ' ', atunci exista zona libera la coordonatele i, j
+	void read_input() {	
+        std::cin >> n >> m;
+        // initial se marcheaza toate elementele din matrice ca fiind zone libere
+        graph = std::vector<std::vector<char>>(n + 1, 
```

Comment:
I think we should keep the convention and replace graph with adj. @neatudarius what do you think?